autopush/tests/test_web_validation.py

```diff
@@ -865,6 +865,99 @@ def test_valid_vapid_crypto_header_webpush(self):
         eq_(errors, {})
         ok_("jwt" in result)
 
+    def test_valid_vapid_02_crypto_header_webpush(self):
+        schema = self._make_fut()
+
+        header = {"typ": "JWT", "alg": "ES256"}
+        payload = {"aud": "https://pusher_origin.example.com",
+                   "exp": int(time.time()) + 86400,
+                   "sub": "mailto:admin@example.com"}
+
+        token, crypto_key = self._gen_jwt(header, payload)
+        auth = "vapid t={token},k={key}".format(token=token,
```

Comment:
I'm not quite sure I'm seeing the problem you've spotted. Draft-02 has two parameters, `t` & `k`. I am missing a test that includes an extra, ignored parameter (and I'll switch parameter order as well), but there's already a test that checks for missing parameter. I switch `k` for a nonsense `n` in a later test that should fail. I'll add comments to the tests to clarify what's being tested.